airbyte-commons-server/src/test/java/io/airbyte/commons/server/handlers/OAuthHandlerTest.java

```diff
@@ -172,15 +182,15 @@ void testBuildJsonPathFromOAuthFlowInitParameters() {
   void testGetOAuthInputConfiguration() {
     final JsonNode hydratedConfig = Jsons.deserialize(
         """
-        {
-          "field1": "1",
-          "field2": "2",
-          "field3": {
-            "field3_1": "3_1",
-            "field3_2": "3_2"
-          }
-        }
-        """);
+            {
```

Comment:
would also double check if this is related to the formatting bits.

Reply:
It was indeed